cryostatio/cryostat3
# fix(jmx): fix regression in JMX auth connection failures

# Welcome to Cryostat3! 👋
## Before contributing, make sure you have:

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Fixes: #438
See #350
See #424

## How to manually test:
1. *Run CRYOSTAT_IMAGE=quay.io... bash smoketest.bash...*
2. Select `vertx-fib-demo-2:9094` and go to Events. The JMX credentials modal should appear very quickly. Close the modal
3. Select `cryostat3:9091`. The JMX credentials modal should appear again. Enter the credentials `cryostat:password`. The modal should disappear and the table should load.
4. Re-select `vertx-fib-demo-2:9094` and this time enter `admin:adminpass123`. The table should reload.
